lib/dns.js

```diff
@@ -62,12 +62,13 @@ function makeAsync(callback) {
   return function asyncCallback() {
     if (asyncCallback.immediately) {
       // The API already returned, we can invoke the callback immediately.
-      callback.apply(null, arguments);
+      callback(...arguments);
```

Comment:
spread is still significantly slower than `apply()`, so we should probably avoid changing it for now.